queue_services/business-pay/src/business_pay/config.py

```diff
@@ -0,0 +1,181 @@
+    "testing": "business_pay.config.TestConfig",
+    "production": "business_pay.config.ProdConfig",
+    "default": "business_pay.config.ProdConfig",
+}
+
+
+def get_named_config(config_name: str = "production"):
+    """Return the configuration object based on the name.
+
+    :raise: KeyError: if an unknown configuration is requested
+    """
+    if config_name in ["production", "staging", "default"]:
+        app_config = ProdConfig()
+    elif config_name == "testing":
+        app_config = TestConfig()
+    elif config_name == "development":
+        app_config = DevConfig()
+    else:
+        raise KeyError(f"Unknown configuration: {config_name}")
+    return app_config
+
+
+class Config:  # pylint: disable=too-few-public-methods
+    """Base class configuration that should set reasonable defaults.
+
+    Used as the base for all the other configurations.
+    """
+
+    PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+    PAYMENT_SVC_URL = os.getenv("PAYMENT_SVC_URL", "")
+
+    SENTRY_DSN = os.getenv("SENTRY_DSN", None)
+
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+    # POSTGRESQL
+    DB_USER = os.getenv("DATABASE_USERNAME", "")
+    DB_PASSWORD = os.getenv("DATABASE_PASSWORD", "")
+    DB_NAME = os.getenv("DATABASE_NAME", "")
+    DB_HOST = os.getenv("DATABASE_HOST", "")
+    DB_PORT = os.getenv("DATABASE_PORT", "5432")
+    # POSTGRESQL
+    if DB_UNIX_SOCKET := os.getenv("DATABASE_UNIX_SOCKET", None):
+        SQLALCHEMY_DATABASE_URI = f"postgresql+pg8000://{DB_USER}:{DB_PASSWORD}@/{DB_NAME}?unix_sock={DB_UNIX_SOCKET}/.s.PGSQL.5432"
+    else:
+        SQLALCHEMY_DATABASE_URI = (
+            f"postgresql+pg8000://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
+        )
+
+    NATS_CONNECTION_OPTIONS = {
+        "servers": os.getenv("NATS_SERVERS", "nats://127.0.0.1:4222").split(","),
+        "name": os.getenv("NATS_CLIENT_NAME", "entity.filing.worker"),
+    }
+    STAN_CONNECTION_OPTIONS = {
+        "cluster_id": os.getenv("NATS_CLUSTER_ID", "test-cluster"),
+        "client_id": str(random.SystemRandom().getrandbits(0x58)),
+        "ping_interval": 1,
+        "ping_max_out": 5,
+    }
+
+    SUBSCRIPTION_OPTIONS = {
+        "subject": os.getenv("NATS_SUBJECT", "entity.filings"),
+        "queue": os.getenv("NATS_QUEUE", "filing-worker"),
+        "durable_name": os.getenv("NATS_QUEUE", "filing-worker") + "_durable",
+    }
+
+    FILER_PUBLISH_OPTIONS = {
+        "subject": os.getenv("NATS_FILER_SUBJECT", "entity.filing.filer"),
+    }
+
+    EMAIL_PUBLISH_OPTIONS = {
+        "subject": os.getenv("NATS_EMAILER_SUBJECT", "entity.email"),
+    }
+
+    ENTITY_FILER_TOPIC = os.getenv("ENTITY_FILER_TOPIC", "filer")
+    ENTITY_MAILER_TOPIC = os.getenv("ENTITY_FILER_TOPIC", "mailer")
+
+    ENVIRONMENT = os.getenv("ENVIRONMENT", "prod")
+
+    GOOGLE_CLIENT_ID = os.getenv("GOOGLE_CLIENT_ID", "google_client_id")
+    GCP_AUTH_KEY = os.getenv("GCP_AUTH_KEY", None)
```

Comment:
`GCP_AUTH_KEY = os.getenv("BUSINESS_GCP_AUTH_KEY`", None)`

Reply:
right, I clean that out!